app/src/main/java/ch/epfl/polychef/fragments/FullRecipeFragment.java

```diff
@@ -227,6 +233,22 @@ public void onClick(View view) {
             }
         });
 
+        Button modifyButton = getView().findViewById(R.id.modifyButton);
+        modifyButton.setOnClickListener(new View.OnClickListener() {
```

Comment:
anonymous class could be replaced by a lambda, you can fix it with Alt+Enter or I'll just take care of it when merging this branch on fix_warnings_TODOs